src/libOpenImageIO/imageinput.cpp

```diff
@@ -1095,6 +1149,107 @@ ImageInput::read_image(int subimage, int miplevel, int chbegin, int chend,
 
 
 
+bool
+ImageInput::read_image(int subimage, int miplevel, int chbegin, int chend,
+                       TypeDesc format, const image_span<std::byte>& data)
+{
+#if 0
```

Comment:
Did you mean to leave this `#if 0` in?

Reply:
Most of the new methods I added are (currently) just having the new API call the old one underneah. This is the one where I did a full implementation to make sure it was working as intended, and I wanted (for now) to have a means of switching back and forth between new and old for debugging purposes.